packager/media/formats/mp4/box_definitions.cc

```diff
@@ -2036,6 +2058,7 @@ size_t AudioSampleEntry::ComputeSizeInternal() {
          sizeof(samplesize) + sizeof(samplerate) + sinf.ComputeSize() +
          esds.ComputeSize() + ddts.ComputeSize() + dac3.ComputeSize() +
          dec3.ComputeSize() + dops.ComputeSize() + dfla.ComputeSize() +
+         dac4.ComputeSize() + mhac.ComputeSize() + alac.ComputeSize() +
          dac4.ComputeSize() + mhac.ComputeSize() + udts.ComputeSize() +
```

Comment:
Hi @cosmin, are `dac4` and `mhac` added twice here?

Reply:
welp, that's embarrassing. Thanks for catching this.